Frameworks/QuartzCore/CALayer.mm

```diff
@@ -1570,9 +1568,9 @@ - (CATransform3D)sublayerTransform {
 */
 - (void)setBackgroundColor:(CGColorRef)color {
     if (color != nil) {
-        priv->backgroundColor = *[static_cast<UIColor*>(color) _getColors];
+        priv->backgroundColor = *(reinterpret_cast<const __CGColorQuad*>(CGColorGetComponents(color)));
```

Comment:
also _backgroundColor is a CGColorRef, so I'm not sure what this is doing.  We should probably just purge QuartzCore of UIColor and __CGColorQuad since they're not the same